app/views/families/show.html.erb

```diff
@@ -1,2 +1,15 @@
-<h1><%= t(".title") %></h1>
-<%= @family.last_name %>
+<p class="title"><%= t(".title") %></p>
+<table>
+  <%= render "families_table_header" %>
+  <tr>
+    <td><%= family.last_name %></td>
+    <td><%= family.first_name %></td>
+    <td><%= family.address %></td>
+    <td><%= family.capability %></td>
+    <td><%= family.preferred_gender %></td>
+    <td><%= family.prof %></td>
+    <td><%= family.bus_driver %></td>
+    <td><%= family.animals %></td>
+    <td><%= family.active %></td>
```

Comment:
If this is the same as what is in the `_family` partial, a suggestion to reuse the partial...

Reply:
Done1